test/zoning-district/zoning-district.repository.mock.ts

```diff
@@ -1,12 +1,41 @@
-import { findByUuidRepoSchema } from "src/zoning-district/zoning-district.repository.schema";
+import {
+  checkByIdRepoSchema,
+  findByUuidRepoSchema,
+  findClassesByIdRepoSchema,
+} from "src/zoning-district/zoning-district.repository.schema";
 import { generateMock } from "@anatine/zod-mock";
 
 export class ZoningDistrictRepositoryMock {
+  numberOfMocks = 1;
+
+  checkZoningDistrictsByIdMocks = Array.from(
+    Array(this.numberOfMocks),
+    (_, seed) => generateMock(checkByIdRepoSchema, { seed: seed + 1 }),
+  );
+
+  async checkZoningDistrictById(id: string) {
+    return this.checkZoningDistrictsByIdMocks.find((row) => row.id === id);
+  }
+
   findByUuidMocks = Array.from(Array(10), () =>
     generateMock(findByUuidRepoSchema, { seed: 1 }),
   );
 
   async findByUuid(id: string) {
     return this.findByUuidMocks.find((row) => row.id === id);
   }
+
+  findClassesByUuidMocks = this.checkZoningDistrictsByIdMocks.map(
+    (checkZoningDistrict) => {
+      return {
+        [checkZoningDistrict.id]: generateMock(findClassesByIdRepoSchema),
+      };
+    },
+  );
+
+  async findClassesByUuid(id: string) {
+    const results = this.findClassesByUuidMocks.find((foo) => id in foo);
+
+    return results === undefined ? results : results[id];
```

Comment:
The real `findClassesByUuid` will return an empty array when nothing matches. The mock would be more accurate to return an empty array, as well.

```suggestion
    return results === undefined ? [] : results[id];
```